docs/reference/flow-yaml-schema-reference.md

```diff
@@ -0,0 +1,222 @@
+| `module`       | string | The module name of the tool using by the node. Used when the `type` is LLM.                                                                                                                                                                               |                                                                                                      |
+| `use_variants` | bool   | Whether the node has variants.                                                                                                                                                                                                                            |                                                                                                      |
+
+
+### Node variants
+Node variants is a dictionary containing variants definition for nodes with variants with their respective node names as dictionary keys.
+Below, we explore the variants for a single node.
+
+| Key                  | Type     | Description                                                                                                                                                                                                                                                                                                   | Allowed values |
+|----------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|
+| `<node_name>`        | string   | The name of the node.                                                                                                                                                                                                                                                                                         |                |
+| `default_variant_id` | string   | Default variant id.                                                                                                                                                                                                                                                                                           |                |
+| `variants `          | object   | This dictionary contains all node variations, with the variant id serving as the key and a node definition dictionary as the corresponding value.  Within the node definition dictionary, the key labeled 'node' should contain a variant definition similar to [Nodes](#nodes), excluding the 'name' field.  |                |
+
+
+
+## Examples
+
+Flow examples are available in the [GitHub repository](https://github.com/microsoft/promptflow/tree/main/examples/flows). 
+Below, we provide a web classification flow example.
```

Comment:
maybe pick some classic example and put directly link here

Reply:
added